src/test/java/uk/nhs/prm/deduction/e2e/models/ContinueRequestMessage.java

```diff
@@ -0,0 +1,43 @@
+package uk.nhs.prm.deduction.e2e.models;
+
+import uk.nhs.prm.deduction.e2e.utility.Resources;
+
+import java.util.UUID;
+
+import static uk.nhs.prm.deduction.e2e.utility.TestUtils.getUUIDAsUpperCaseString;
+
+public class ContinueRequestMessage {
+    private UUID conversationId;
+    private UUID messageId;
+    private String sourceGpOds;
+    private String destinationGpOds;
+    private String sourceGpAsid;
+    private String destinationGpAsid;
+
+    public ContinueRequestMessage(UUID conversationId, UUID messageId, String sourceGpOds, String destinationGpOds, String sourceGpAsid, String destinationGpAsid) {
+        this.conversationId = conversationId;
+        this.messageId = messageId;
+        this.sourceGpOds = sourceGpOds;
+        this.destinationGpOds = destinationGpOds;
+        this.sourceGpAsid = sourceGpAsid;
+        this.destinationGpAsid = destinationGpAsid;
+    }
```

Comment:
Please leverage Lombok here in favour of adding the following annotation to the class level :)

```
@AllArgsConstructor
```

Reply:
Thanks!  It's so good to know this useful annotation! Boilerplate gone now :D